mage_ai/frontend/interfaces/IntegrationSourceType.ts

```diff
@@ -137,6 +137,7 @@ export enum IntegrationSourceEnum {
 
 export enum IntegrationDestinationEnum {
   BIGQUERY = 'bigquery',
+  CLICKHOUSE = 'clickhouse',
```

Comment:
no need to update frontend code here. we get destination list from backend.

Reply:
removed on d27c638831d274c104381dcc25fde217b37e0a86